### PR TITLE
Add Query String for rollups

### LINKED ISSUE #2341: [DOC] Query String Query support in Rollups
**What do you want to do?**
 
- [ ] Request a change to existing documentation

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
 ISM currently supports multiple types of search queries in Rollups such as Term, Terms, Range, etc. We want to now add support for [Query String](https://opensearch.org/docs/latest/opensearch/query-dsl/full-text/#query-string) (https://opensearch.org/docs/latest/opensearch/query-dsl/full-text/#query-string) search queries in Rollups.

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.

### PR DESCRIPTION
**Pull request by Naarcha-AWS**
Signed-off-by: Naarcha-AWS <naarcha@amazon.com>

### Description

Relates to #2427. 

Fixes #2341. 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _im-plugin/index-rollups/index.md [hunk 1]
```diff
@@ -518,6 +518,57 @@ POST example_rollup/_search
 
 The `doc_count` field in bucket aggregations contains the number of documents collected in each bucket. When calculating the bucket's `doc_count`, the number of documents is incremented by the number of the pre-aggregated documents in each summary document. The `doc_count` returned from rollup searches represents the total number of matching documents from the source index. Thus, the document count for each bucket is the same whether you search the source index or the rollup target index.
 
+## Query string queries
+
+o take advantage of shorter and easier to write strings in Query DSL, you can use [query strings]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/query-string/) to simplify search queries in rollup indexes. To use query strings, add the following fields to your rollup search request.
```
Comment (kolchfa-aws) [inline _im-plugin/index-rollups/index.md:523]:
```suggestion
To take advantage of shorter and easier to write strings in Query DSL, you can use [query strings]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/query-string/) to simplify search queries in rollup indexes. To use query strings, add the following fields to your rollup search request.
```

**Review by kolchfa-aws — APPROVED** [review]
LGTM; one typo fix.

**Review by ariamarble — APPROVED** [review]
LGTM

File: _im-plugin/index-rollups/index.md [hunk 2]
```diff
@@ -518,6 +518,57 @@ POST example_rollup/_search
 
 The `doc_count` field in bucket aggregations contains the number of documents collected in each bucket. When calculating the bucket's `doc_count`, the number of documents is incremented by the number of the pre-aggregated documents in each summary document. The `doc_count` returned from rollup searches represents the total number of matching documents from the source index. Thus, the document count for each bucket is the same whether you search the source index or the rollup target index.
```
Comment (carolxob) [inline _im-plugin/index-rollups/index.md:519]:
```suggestion
The `doc_count` field in bucket aggregations contains the number of documents collected in each bucket. When calculating the bucket's `doc_count`, the number of documents is incremented by the number of the pre-aggregated documents in each summary document. The `doc_count` returned from rollup searches represents the total number of matching documents from the source index. The document count for each bucket is the same whether you search the source index or the rollup target index.
```

**Pull request merged**